agent/src/main/java/com/criteo/hadoop/garmadon/agent/tracers/MethodTracer.java

```diff
@@ -5,9 +5,9 @@
 import net.bytebuddy.implementation.Implementation;
 import net.bytebuddy.matcher.ElementMatcher;
 
-abstract class MethodTracer extends Tracer {
+public abstract class MethodTracer extends Tracer {
```

Comment:
plz don't change visibility

Reply:
How can I do so if I move Tracers in sub package which I think is more readable?

Reply:
Ok I did not see the change of package and I know that intellij proposes protected instead of package-protected. I thought that was what happened, an auto correct from intellij. You can forget my comments about visibility